From d7496f873617892ce82b200c3acdf3d87141d06c Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@digia.com>
Date: Thu, 9 May 2013 15:46:15 +0300
Subject: [PATCH] help python find itself correctly

The WITH_PYTHON_PATH python path does not work with relocatable
SDK. Instead use program name since it's bound to be in correct place.
---
 gdb/main.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/gdb/main.c b/gdb/main.c
index 976f8ae..aa1913d 100644
--- a/gdb/main.c
+++ b/gdb/main.c
@@ -369,10 +369,7 @@ captured_main (void *data)
 #ifdef WITH_PYTHON_PATH
   {
     /* For later use in helping Python find itself.  */
-    char *tmp = concat (WITH_PYTHON_PATH, SLASH_STRING, "lib", NULL);
-
-    python_libdir = relocate_gdb_directory (tmp, PYTHON_PATH_RELOCATABLE);
-    xfree (tmp);
+    python_libdir = gdb_program_name;
   }
 #endif
 
-- 
1.7.10.4

